src/config.c

```diff
@@ -750,6 +754,33 @@ short load_configuration(void)
                 COMMAND_TEXT(cmd_num),config_textname);
           }
           break;
+	  case 15: // Atmos
+          for (i=0; i<3; i++)
+          {
+            if (get_conf_parameter_single(buf,&pos,len,word_buf,sizeof(word_buf)) > 0)
+              k = atoi(word_buf);
+            else
+              k = -1;
+            if (k<=0)
+            {
+                CONFWRNLOG("Couldn't recognize setting %d in \"%s\" command of %s file.",
```

Comment:
@AdamPlenty This doesn't seem to work for me. I expect to get this message when I don't have the setting in the log, but I never see it.

Reply:
> 
> 
> @AdamPlenty This doesn't seem to work for me. I expect to get this message when I don't have the setting in the log, but I never see it.

I don't know why that would be. Does it work when you have no resolution settings?

Reply:
It does not, no.